types/model.d.ts

```diff
@@ -19,17 +19,17 @@ export type PropertyType =
  * Property definition
  */
 export interface PropertyDefinition extends AnyObject {
-  name: string;
```

Comment:
AFAICT, the property name is always stored as the key in property map (name => definition).